.github/workflows/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,17 @@
+blank_issues_enabled: false
+contact_links:
+  - name: New Idea?
+    url: https://github.com/CommunityToolkit/Labs-Windows/discussions?discussions_q=category%3AExperiments+category%3AIdeas
+    about: Labs is where we propose and experiment with new ideas. Search ongoing experiments or contribute your own!
+  - name: Questions about the Toolkit?
+    url: https://github.com/CommunityToolkit/Windows/discussions/categories/questions-help
```

```suggestion
    url: https://github.com/CommunityToolkit/Windows/discussions/categories/q-a
```
The Q&A category is the one we have in this repo.